starter-code/MyLinkedList.java

```diff
@@ -0,0 +1,119 @@
+
+public class MyLinkedList<T>{
+
+	private Node<T> head;
+	private int size = 0;
+
+	public int getSize() {
+		return size;
+	}
+
+	public boolean remove(int index) {
+        if(index < 0 || index > size) {
+            return false;
+        }
+        Node<T> current = head;
+
+        if(head != null) {
+            while(index > 0) {
+                if(current.getNext() == null) {
+                    return false;
+                }
+                current = current.getNext();
+                index--;
+            }
+            current.setNext(current.getNext().getNext());
+            size--;
+            return true;
+        }
+        return false;
+	}
+
+	public T get(int index) {
+        if(index < 0 || index > size) {
+            return null;
+        }
+
+        Node<T> current = head;
+
+        if(head != null) {
+            current = current.getNext();
+            for(int i=0;i<index;i++) {
+                if(current.getNext() == null) {
+                    return null;
+                }
+                current = current.getNext();
+            }
+            return current.getData();
+        }
+        return null;
+	}
+
+	public void add(T obj) {
+		if(head == null) {
+            head = new Node<>(obj);
+        }
+
+        Node<T> added = new Node<>(obj);
+        Node<T> current = head;
+
+        if(current != null) {
+            while(current.getNext() != null) {
+                current = current.getNext();
+            }
+            current.setNext(added);
+        }
+        size++;
+	}
+
+	public void reverseList() {
+        Node<T> node = head;
+        Node<T> last = null;
+        Node<T> current = node;
+        Node<T> next;
+        while(current != null) {
+            next = current.getNext();
+            current.next = last;
+            last = current;
+            current = next;
+        }
+        node = last;
+        head = node;
```

Comment:
I think you can do all this without the `node` local variable, but this looks good :)